src/Microsoft.DotNet.ProjectModel/ProjectReader.cs

```diff
@@ -132,6 +132,37 @@ public Project ReadProject(Stream stream, string projectName, string projectPath
                     throw new FormatException("The assembly file version is invalid: " + fileVersion, ex);
                 }
             }
+            
+            var analyzerOptionsJson = rawProject.Value("analyzerOptions") as JsonObject;
```

Comment:
Should this be a child node of compilationOptions? Not sure we want another top level node?